SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -5,104 +5,230 @@
     using System.Net;
     using System.Net.Http;
     using System.Threading;
     using System.Threading.Tasks;
     using BoDi;
     using Client;
+    using Ductus.FluentDocker;
     using Ductus.FluentDocker.Builders;
+    using Ductus.FluentDocker.Common;
     using Ductus.FluentDocker.Model.Builders;
     using Ductus.FluentDocker.Services;
     using Ductus.FluentDocker.Services.Extensions;
     using OpenQA.Selenium;
     using OpenQA.Selenium.Chrome;
+    using Shared.Logger;
     using TechTalk.SpecFlow;
 
-    public class DockerHelper
+    public class DockerHelper : Shared.IntegrationTesting.DockerHelper
     {
-        public INetworkService TestNetwork;
+        private static void AddEntryToHostsFile(String ipaddress, String hostname)
+        {
+            if (FdOs.IsWindows())
+            {
+                using (StreamWriter w = File.AppendText(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.System), @"drivers\etc\hosts")))
+                {
+                    w.WriteLine($"{ipaddress} {hostname}");
+                }
+            }
+            else if (FdOs.IsLinux())
+            {
+                ExecuteBashCommand($"echo {ipaddress} {hostname} | sudo tee -a /etc/hosts");
+            }
+        }
 
-        public IContainerService SecurityServiceTestUIContainer;
+        static string ExecuteBashCommand(string command)
+        {
+            // according to: https://stackoverflow.com/a/15262019/637142
+            // thans to this we will pass everything as one command
+            command = command.Replace("\"", "\"\"");
 
-        public String SecurityServiceTestUIContainerName;
+            var proc = new Process
+                       {
+                           StartInfo = new ProcessStartInfo
+                                       {
+                                           FileName = "/bin/bash",
+                                           Arguments = "-c \"" + command + "\"",
+                                           UseShellExecute = false,
+                                           RedirectStandardOutput = true,
+                                           CreateNoWindow = true
+                                       }
+                       };
+            Console.WriteLine(proc.StartInfo.Arguments);
 
-        public ISecurityServiceClient SecurityServiceClient;
-        
+            proc.Start();
+            proc.WaitForExit();
+
+            return proc.StandardOutput.ReadToEnd();
+        }
+
+        private static IContainerService SetupSecurityServiceContainer(String containerName,
+                                                                      ILogger logger,
+                                                                      String imageName,
+                                                                      INetworkService networkService,
+                                                                      String hostFolder,
```

Comment:
Codacy found an issue: [Remove this unused method parameter 'hostFolder'.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)